README.md

```diff
@@ -1,48 +1,87 @@
+>
+> Open the `Advanced...` box and check the `Run as administrator` checkbox.
+>
+> Save and close.
 
-This should be considered BETA - it is working well for me, but at the time of writing this no one else has tested it so far.
+- If you can't access the sunshine stream for whatever reason, you can try a couple of things:
+    - Open a Windows Terminal and run this command:
+    ```batch
+    DisplaySwitch 1
+    ```
 
-While I'm pretty confident this will not break your computer, I don't know enough about Windows Drivers to be 100% sure.  Further, there is a real risk that your displays won't come back after a streaming sesion if there is some issue with Sunshine or the scripts - during development and testing I hit quite a few of those cases; I'm confident it shouldn't happen during normal operation anymore, but just in case it does, you can escape in a few ways:
+> [!NOTE]
+> This should enable only your primary display, which should not be the virtual monitor, allowing you to fix things up.
 
-- If your displays don't come back, but sunshine is still running, you can get back in the stream and fix things up:
-    - Disable the display device - you can run the command `pnputil /disable-device /deviceid root\iddsampledriver` from a privileged terminal.  If you want to be extra secure you can try to bind this to some key combination ahead of time (make sure it runs as admin).
-- If you can't access the sunshine stream for whatever reason, you can try a couple of things:
-    - Pres windows key + write "terminal" + press Enter + write "DisplaySwitch 1" + press Enter.  This should enable only your primary display, which should not be the virtual monitor, allowing you to fix things up.
-    - Press windows + P to get into the display selection dialogue, then tab + move around to select something different from the current setup.  I recommend practicing this ahead of time if you want to go this route, just so you have an idea of what it feels like; in the broken case it should have the "second screen only" option pre-selected.
+-
+    - Press `Windows + P` to get into the display selection dialogue, then press `Tab` + move around to select something different from the current setup. I recommend practicing this ahead of time if you want to go this route, just so you have an idea of what it feels like; in the broken case it should have the "second screen only" option pre-selected.
+    
     - Connect another display to your computer - this will cause windows to try and apply a new configuration to the displays that should get signal on at least one of the real ones.
+    
     - If you already have a second monitor, disconnect it - similar to the point above, might result in getting signal back.
 
-## Dependencies
+## Setup
+
+First, download the [latest release](https://github.com/LeGeRyChEeSe/sunshine-virtual-monitor/releases/latest) (`.zip` file) and unzip it.
```

Comment:
```suggestion
First, download the [latest release](https://github.com/Cynary/sunshine-virtual-monitor/releases/latest) (`.zip` file) and unzip it.
```

When you will be good to merge, don't forget to modify this line **before** merging.
Then push a new release so the link could point to the actual latest release (e.g. Push a release with a new `v1.0.0` tag **after** you merged this branch to the `main`)

Or you can also remove this line and add a new line to request the user to download the code directly, or to clone it with git.

Reply:
Will do, thanks!